CONFIGDOC.md

```diff
@@ -1,5 +1,6 @@
 When you define an automated pipeline of tasks in Jenkins&trade;, whether in the web UI or with a [`Jenkinsfile`](https://www.jenkins.io/doc/book/pipeline/jenkinsfile/), you can use the plugin to run your MATLAB&reg; code or Simulink&reg; models. This guide demonstrates how to configure the plugin and use it in freestyle, multi-configuration, and Pipeline projects.
 
+> :information_source: **Note:** To run MATLAB code or Simulink models, Jenkins requires a valid MATLAB or Simulink license. If you have installed Jenkins as a Windows&reg; service application, you need to provide a MathWorks&reg; username and password in the **Jenkins Properties** window (accessible from within the Windows Task Manager).

```

Comment:
Just to confirm, is it a MathWorks username/password or is it supposed to be a Windows username/password? 🤔

Reply:
its uer name and password who is authrnticated to run MATLAB

Reply:
Probably we can say Jenkins Service Properties

Reply:
@abhargav-mw, do you mean we update "Jenkins Properties" to "Jenkins Service Properties"? The reason I am a bit hesitant is that we don't want to modify the text in the UI.